### PR TITLE
Fix component auto import

### PR DESCRIPTION
**Pull request by sebastianbenz**
This fixes the problem that the validation rules include experimental
component versions causing bento components to be auto imported.

Instead of generating the component versions from validation rules, we
extract the latest version from the component docs.

This PR also removes the buildComponentVersion task as it is no longer
needed.

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@matthiasrohmer build is passing now, not super happy about commiting the component versions file, but I don't see another way.

File: pages/content/amp-dev/about/websites-2021.html [hunk 1]
```diff
@@ -112,7 +112,7 @@ <h2 class="ap-h1">{{ _('Grab-and-go components') }}</h2>
         <p>
           {{ _('Want just the components without the AMP ecosystem? ')}}
           <a
-            href="{{ g.doc('/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide', locale=doc.locale).url.path }}"
+            href="{{ g.doc('../documentation/guides-and-tutorials/start/bento_guide.md', locale=doc.locale).url.path }}"
```
Comment (patrickkettner) [inline pages/content/amp-dev/about/websites-2021.html:115]: im assuming relative URLs don't break anything?

File: pages/extensions/amp_example_preview/util/example_document_test.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def test_different_attributes(self):
     self.assertEqual('none', example_doc.preview)
     self.assertEqual(1, len(example_doc.imports))
     self.assertEqual('amp-accordion', example_doc.imports[0].name)
-    self.assertEqual('1.0', example_doc.imports[0].version) # 0.1 is always the default
```
Comment (patrickkettner) [inline pages/extensions/amp_example_preview/util/example_document_test.py:46]: lol

File: pages/static/files/component-versions.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
+{
```
Comment (patrickkettner) [inline pages/static/files/component-versions.json:1]: is this just going to be manually updated? What calls componentReferenceImporter to update it?

**Review by patrickkettner — APPROVED** [review]
not clear on some stuff that I ask about, but looks good!

File: pages/static/files/component-versions.json (hunk 3 above)
Comment (matthiasrohmer) [reply pages/static/files/component-versions.json:1]: `componentReferenceImporter` is called with with every build and by running `npx gulp bootstrap` or `npx gulp importAll` for devs running amp.dev locally.

**Comment by patrickkettner** [comment]
just as a devil's advocate, did this issue get discussed with the validator folks? I assume they had no workarounds?

File: pages/content/amp-dev/about/websites-2021.html (hunk 1 above)
Comment (matthiasrohmer) [reply pages/content/amp-dev/about/websites-2021.html:115]: Yeah that should be fine.

**Review by matthiasrohmer — APPROVED** [review]
Hm. I liked having the validator rules as a single point of truth but they have always been tedious to work with... no better idea.

**Pull request merged**